lib/controllers/deploy-controller.ts

```diff
@@ -11,7 +11,11 @@ export class DeployController {
 
 		const executeAction = async (device: Mobile.IDevice) => {
 			const deviceDescriptor = _.find(deviceDescriptors, dd => dd.identifier === device.deviceInfo.identifier);
-			await this.$prepareController.prepare(deviceDescriptor.buildData);
+			const prepareData = {
+				...deviceDescriptor.buildData,
+				nativePrepare: { skipNativePrepare: !!deviceDescriptor.skipNativePrepare }
```

Comment:
Why is the `!!` necessary? What are the possible values of deviceDescriptor.skipNativePrepare?

Reply:
!! converts the value to boolean, it's just a protection in case it is passed as undefined.